src/Files.App/Utils/Shell/Win32API.cs

```diff
@@ -387,6 +387,10 @@ public static async Task<bool> RunPowershellCommandAsync(string command, bool ru
 			{
 				return false;
 			}
+			catch (InvalidOperationException)
+			{
+				return false;
+			}
```

Comment:
Since this exception should not normally occur, it is better to log the stack trace for debugging purposes.

Reply:
Should we also throw?

Reply:
I don't think so. Just logging is enough.